### PR TITLE
Allow metadata

### PR DESCRIPTION
**Pull request by wbowling**
For required purchasing clips on certain subscriptions

### CODE REVIEW

File: lib/shutterstock-ruby/videos.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ def details(id, options = {})
       JSON.parse(get('/videos', { id: id }.merge(options)))
     end
 
-    def purchase(id, subscription_id, size, options = {})
+    def purchase(id, subscription_id, size, options = {}, metadata = {})
```
Comment (wbowling) [inline lib/shutterstock-ruby/videos.rb:13]: I would have liked to have `metadata` before `options` but that would be backwards incompatible

Comment (dnahodil) [reply lib/shutterstock-ruby/videos.rb:13]: Would it make sense for `metadata` to be an entry in `options`?

Comment (wbowling) [reply lib/shutterstock-ruby/videos.rb:13]: Yeah it could, but we don't want it to be passed to `post` as an option

Comment (dnahodil) [reply lib/shutterstock-ruby/videos.rb:13]: Oh yeah, I see.

**Comment by dnahodil** [comment]
Looks okay to me, and it's running in prod so I guess 🛒

**Comment by jkburges** [comment]
How can it be in prod, if it's not merged here?

@dnahodil, would you mind taking responsibility for this, since Will is away? (since you know slightly more about it than I do :-) )

**Comment by dnahodil** [comment]
> How can it be in prod, if it's not merged here?

GemFile is pointing to the branch

> @dnahodil, would you mind taking responsibility for this, since Will is away?

I... uh... um... [okay](http://i0.kym-cdn.com/entries/icons/mobile/000/003/617/OkayGuy.jpg).

**Comment by jkburges** [comment]
> GemFile is pointing to the branch

Correct, was just testing you.

> I... uh... um... okay.

You don't have to, I just thought that you knew about this.  And you are the gopher.

**Comment by dnahodil** [comment]
> You don't have to

I'll do it if I get time. 👍  It means merging this PR and updating the web-app GemFile.

**Comment by wbowling** [comment]
Closed https://github.com/Biteable/shutterstock-ruby/pull/6

**Pull request closed without merging**